### PR TITLE
[Security] Added missing translations for Croatian (hr)

### LINKED ISSUE #41039: Missing translations for Croatian (hr)
Hello,

There are some missing translation for Croatian (hr). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by HypeMC**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41039
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Link za prijavu je isteako ili je neispravan.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Previše neuspjelih pokušaja prijave, molim pokušajte ponovo za %minutes% minutu.</target>
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Previše neuspjelih pokušaja prijave, molim pokušajte ponovo za %minutes% minuta.</target>
```
Comment (HypeMC) [inline src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: Note: this is actually wrong as it should be 2..4 minut**e**, 5..20 minut**a**, 21 minut**u**, 22..24 minut**e**, etc., but I don't think that's currently possible.

Comment (OskarStark) [reply src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: IIRC we had a PR with this "problem" one or two months ago. Maybe you can double check the recent closed/open PRs.

@Nyholm do you remember?

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: I think you are referring to this PR: https://github.com/symfony/symfony/pull/41097

@HypeMC, do you see how they achieved that for Lithuanian?

Comment (OskarStark) [reply src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: Exactly, many thanks Tobias 🌴

Comment (HypeMC) [reply src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: @OskarStark @Nyholm Works like a charm, thanks for the help.

Comment (OskarStark) [reply src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf]: Very welcome, thanks for another great contribution 🥰

**Review by OskarStark — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @HypeMC.

**Pull request merged**